README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+Hey there!  My name is Kristina Wald.  Here are some quick facts about me:
+
+# Age
+25
+
+# Where I'm from
+Hyde Park - *grew up practically right on campus!*
+
+# What I'm studying
+Behavioral Science at Booth
+
+# Research interests
+* Person perception
+* Impression formation
+* Social interaction
+
+# Favorite things to do
+* Take walks *(especially along Lake Michigan!)*
+* Watch _**The Bachelor**_
+* Sleep
+
+# Fun Facts
```

Comment:
One of my friends also likes to go to Starbucks and she only drinks non caffeinated frappuccino